extensions/warp-ipfs/src/store/conversation.rs

```diff
@@ -624,26 +623,26 @@ impl ConversationDocument {
     }
 }
 
-impl From<ConversationDocument> for Conversation {
-    fn from(document: ConversationDocument) -> Self {
-        Conversation::from(&document)
-    }
-}
-
-impl From<&ConversationDocument> for Conversation {
-    fn from(document: &ConversationDocument) -> Self {
-        let mut conversation = Conversation::default();
-        conversation.set_id(document.id);
-        conversation.set_name(document.name.clone());
-        conversation.set_creator(document.creator.clone());
-        conversation.set_conversation_type(document.conversation_type);
-        conversation.set_recipients(document.recipients());
-        conversation.set_created(document.created);
-        conversation.set_settings(document.settings);
-        conversation.set_modified(document.modified);
-        conversation
-    }
-}
+// impl From<ConversationDocument> for Conversation {
+//     fn from(document: ConversationDocument) -> Self {
+//         Conversation::from(&document)
+//     }
+// }
+
+// impl From<&ConversationDocument> for Conversation {
+//     fn from(document: &ConversationDocument) -> Self {
+//         let mut conversation = Conversation::default();
+//         conversation.set_id(document.id);
+//         conversation.set_name(document.name.clone());
+//         conversation.set_creator(document.creator.clone());
+//         conversation.set_conversation_type(document.conversation_type);
+//         conversation.set_recipients(document.recipients());
+//         conversation.set_created(document.created);
+//         conversation.set_settings(document.settings);
+//         conversation.set_modified(document.modified);
+//         conversation
+//     }
+// }
```

Comment:
This doesn't make any sense after `Conversation` becomes a trait so there is no reason to keep it commented out. Just delete it.

Reply:
Left there unintentionally